agent/lib/internal_addressplan_source.js

```diff
@@ -89,9 +89,7 @@ function same_addressplan_resources(a, b) {
 
 function same_ttl(a, b) {
```

Comment:
Maybe reuse function from internal_address_source

Reply:
Hopefully I've addressed this criticism in the latest code.